core/Forms/Forms.php

```diff
@@ -31,4 +34,66 @@ private function checkInput($data) {
         return htmlentities($data);
     }
 
+    public function notEmpty($field) {
+        return empty($field);
+    }
+    public function isEmpty($field) {
```

Comment:
Are you sure that this is semantically correct function isEmpty returns if string if not empty?